test/UID2.Client.Test.Utils/UID2TokenGenerator.cs

```diff
@@ -99,7 +127,7 @@ public static string GenerateEUIDTokenV3(string uid, Key masterKey, int siteId,
         /// <param name="siteKey">site-specific key to encrypt the UID with first before encrypting again with master key</param>
         /// <param name="encryptParams"></param>
         /// <returns>the encrypted UID in the form of UID2 Token</returns>
-        public static string GenerateUID2TokenV3(string uid, Key masterKey, int siteId, Key siteKey, Params encryptParams)
+        private static string GenerateUID2TokenWithDebugInfo(string uid, Key masterKey, int siteId, Key siteKey, Params encryptParams, bool v4AdToken)
```

Comment:
consider using enum instead of bool parameter :)

Reply:
fixed